biodatasets/pubtator_central/pubtator_central.py

```diff
@@ -0,0 +1,322 @@
+    PLoS ONE,7(6):e38460
+[6] Wei C-H, et al., (2017) "Integrating genomic variant information from literature with dbSNP and
+    ClinVar for precision medicine", Bioinformatics,34(1): 80-87
+"""
+
+
+from typing import Dict, Iterator, List, Tuple
+
+import datasets
+from bioc import pubtator
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@article{10.1093/nar/gkz389,
+  title        = {{PubTator central: automated concept annotation for biomedical full text articles}},
+  author       = {Wei, Chih-Hsuan and Allot, Alexis and Leaman, Robert and Lu, Zhiyong},
+  year         = 2019,
+  month        = {05},
+  journal      = {Nucleic Acids Research},
+  volume       = 47,
+  number       = {W1},
+  pages        = {W587-W593},
+  doi          = {10.1093/nar/gkz389},
+  issn         = {0305-1048},
+  url          = {https://doi.org/10.1093/nar/gkz389},
+  eprint       = {https://academic.oup.com/nar/article-pdf/47/W1/W587/28880193/gkz389.pdf}
+}
+"""
+
+_DATASETNAME = "pubtator_central"
+
+_DESCRIPTION = """\
+PubTator Central (PTC, https://www.ncbi.nlm.nih.gov/research/pubtator/) is a web service for
+exploring and retrieving bioconcept annotations in full text biomedical articles. PTC provides
+automated annotations from state-of-the-art text mining systems for genes/proteins, genetic
+variants, diseases, chemicals, species and cell lines, all available for immediate download. PTC
+annotates PubMed (30 million abstracts), the PMC Open Access Subset and the Author Manuscript
+Collection (3 million full text articles). Updated entity identification methods and a
+disambiguation module based on cutting-edge deep learning techniques provide increased accuracy.
+"""
+
+_HOMEPAGE = "https://www.ncbi.nlm.nih.gov/research/pubtator/"
+_LICENSE = """\
+PUBLIC DOMAIN NOTICE
+National Center for Biotechnology Information
+
+This software/database is a "United States Government Work" under the terms of the United States Copyright Act.
+It was written as part of the authors' official duties as a United States Government employee and thus cannot be
+copyrighted. This software/database is freely available to the public for use. The National Library of Medicine
+and the U.S. Government have not placed any restriction on its use or reproduction.
+
+Although all reasonable efforts have been taken to ensure the accuracy and reliability of the software and data,
+the NLM and the U.S. Government do not and cannot warrant the performance or results that may be obtained by
+using this software or data. The NLM and the U.S. Government disclaim all warranties, express or implied,
+including warranties of performance, merchantability or fitness for any particular purpose.
+"""
+
+_URLS = {_DATASETNAME: "https://ftp.ncbi.nlm.nih.gov/pub/lu/PubTatorCentral/bioconcepts2pubtatorcentral.offset.sample"}
```

Comment:
will probably want one url pointing to the sample and one url pointing to the full dataset